lib/src/analyzers/lint_analyzer/rules/rules_list/format_comment/visitor.dart

```diff
@@ -0,0 +1,60 @@
+part of 'format_comment_rule.dart';
+
+const commentsOperator = {
+  CommentType.base: '//',
+  CommentType.documentation: '///',
+  CommentType.multiline: '/*',
+};
+
+class _Visitor extends RecursiveAstVisitor<void> {
+  final _comments = <CommentInfo>[];
+
+  Iterable<CommentInfo> get comments => _comments;
+
+  void checkComments(AstNode node) {
```

Comment:
Why not to use `visitComment` from `RecursiveAstVisitor` instead?

Reply:
Because RecursiveAstVisitor is not found in all nodes. This search method recommended for me by @dkrutskikh